src/commands/ecr-login.yml

```diff
@@ -19,8 +27,5 @@ steps:
   - run:
       name: Log into Amazon ECR
       command: |
-        # aws ecr get-login returns a login command w/ a temp token
-        LOGIN_COMMAND=$(aws ecr get-login --no-include-email --region $<<parameters.region>> --profile <<parameters.profile-name>>)
-
-        # save it to an env var & use that env var to login
-        $LOGIN_COMMAND
+        # get-login-password returns a password that we pipe to the docker login command
+        aws ecr get-login-password --region $<<parameters.region>> --profile <<parameters.profile-name>> | docker login --username AWS --password-stdin <<parameters.account-url>>
```

Comment:
I gave this a run and saw that this integration test did not succeed: https://app.circleci.com/pipelines/github/CircleCI-Public/aws-ecr-orb/227/workflows/a6ed990e-22b7-4cbe-9a0a-ef00e941b2f0/jobs/640

Could you add a `$` in front of `<<parameters.account-url>>`?